includes/admin/class-wcmp-export-consignments.php

```diff
@@ -337,8 +335,23 @@ private function getInsurance(): int
      */
     private function getInsuranceAmount($isInsuranceActive)
     {
+        // Checks if all parcels must be insured
         if ($isInsuranceActive) {
-            return $this->getSetting("{$this->carrier}_" . WCMP_Settings::SETTING_CARRIER_DEFAULT_EXPORT_INSURED_AMOUNT);
+            // get min price for insurance
+            $insuranceFromPrice = (float) $this->getSetting("{$this->carrier}_" .
+                WCMP_Settings::SETTING_CARRIER_DEFAULT_EXPORT_INSURED_FROM_PRICE
+            );
+
+            $insuranceMaxPrice = (int) $this->getSetting("{$this->carrier}_" .
```

Comment:
Is dit int of float?

Reply:
Dit is een float, wordt geselecteerd met een dropdown in de frontend. Geen kommagetallen.